backend/account_usage/models.py

```diff
@@ -0,0 +1,47 @@
+import uuid
+
+from django.db import models
+
+
+class PageUsage(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
+        default=uuid.uuid4,
+        editable=False,
+        db_comment="Primary key for the usage entry, automatically generated UUID",
+    )
+
+    organization_id = models.CharField(
+        default="mock_org",
+        null=False,
+        blank=False,
+    )
+    file_name = models.CharField(
```

Comment:
NIT: Might be a good idea to store the entire file path in case its read from a file system

Reply:
@chandrasekharan-zipstack 
even the file name does not make much sense if we are running the tools in the container( During workflow execution the fle_name will always be `INFILE`)
In a similar view, the file path also won't add much sense. 

Only place it make send is when we do file processing from prompt studio side.

Reply:
@johnyrahul got it - I lacked context of what this field was used for. Somehow it might be good to indicate (preferably through the name or a docstring / comment) that we store the mangled / copied file path. Reason being there might be cases where we'd like to display the user provided file path somewhere and such fields won't be of help to us
If it doesn't hurt us - we could store both user provided file path and the file path that we store during execution